src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String password;
+    private byte[] salt;
+
+    @Column(unique = true)
+    private String email;
+
+    public User() {
+    }
+
+    public User(String email, String password) {
+        this.password = password;
+        this.email = email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public void setSalt(byte[] salt) {
+        this.salt = salt;
+    }
+
+    public byte[] getSalt() {
+        return salt;
+    }
+
+    @Override
+    public String toString() {
+        return "User{ " + "password='" + password + '\''
```

Comment:
I think it's not safe to add password field in toString() method

Reply:
done